CMakeLists.txt

```diff
@@ -242,8 +242,14 @@ set(CMAKE_DEBUG_POSTFIX "" CACHE STRING "Generate debug library name with a post
 mark_as_advanced(CMAKE_DEBUG_POSTFIX)
 
 # Configure find_package behavior
-if(NOT DEFINED CMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY)
-  set(CMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY 1)
+if("${CMAKE_VERSION}" VERSION_GREATER_EQUAL "3.16")
+  if(NOT DEFINED CMAKE_FIND_USE_PACKAGE_REGISTRY)
+    set(CMAKE_FIND_USE_PACKAGE_REGISTRY 1)
```

Comment:
I thought the logic here is positive, so this should be `FALSE`, not `1`.

Reply:
An other good catch ... :pray:  Thanks for the review